wordpress-mobile/gutenberg-mobile
# Implement copy/paste of images in the file/binary format

Fixes #730 

Related Gutenberg PR: https://github.com/WordPress/gutenberg/pull/14802

This PR implements the copy paste of images on the iOS side.
It works by intercepting the paste method on Aztec and passing a url path  array to the JS side to import.

It also implements delegate methods to ask the native, host app to import them in any way it wants.
On the demo app we are just simulating an upload, but on the main WP app we should import the media to the site.

To test:
 - Open the demo app
 - Open the Photos app
- Select one or more images
- Tap on the copy button
- Go back to the demo app
- Select a text block, and tap paste
- See that the images are inserted in the page.

@mzorz do you want to add the android implementation if needed? Can you do copy/paste of a binary image in Android?
